roqollage/tests/interface.rs

```diff
@@ -40,8 +40,8 @@ use test_case::test_case;
 #[test_case(Operation::from(FSwap::new(0, 1)); "FSwap")]
 #[test_case(Operation::from(MeasureQubit::new(0,"ro".to_owned(), 0)); "MeasureQubit")]
 #[test_case(Operation::from(Toffoli::new(0, 1, 2)); "Toffoli")]
-#[test_case(Operation::from(GateDefinition::new(vec![Operation::from(RotateX::new(0, CalculatorFloat::from("theta"))), Operation::from(RotateX::new(1, CalculatorFloat::PI))].into_iter().collect(), "test_gate".to_owned(), vec![0, 1], vec!["theta".to_owned()])); "GateDefinition")]
-#[test_case(Operation::from(CallDefinedGate::new("test".to_owned(), vec![0, 1], vec![CalculatorFloat::from("3.14")])); "CallDefinedGate")]
+// #[test_case(Operation::from(GateDefinition::new(vec![Operation::from(RotateX::new(0, CalculatorFloat::from("theta"))), Operation::from(RotateX::new(1, CalculatorFloat::PI))].into_iter().collect(), "test_gate".to_owned(), vec![0, 1], vec!["theta".to_owned()])); "GateDefinition")]
```

Comment:
Why has this been commented out?

Reply:
It was moved in a feature, I created a test for it now